protocol/src/main.ts

```
@@ -188,6 +192,14 @@ Bun.serve({
       }
     }
 
+    // Bull Board UI at /dev/queues
+    if (url.pathname === '/dev/queues' || url.pathname.startsWith('/dev/queues/')) {
+      const res = await adminQueuesApp.fetch(req);
+      const newHeaders = new Headers(res.headers);
+      Object.entries(corsHeaders).forEach(([key, value]) => newHeaders.set(key, value));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)